controllers/user.js

```diff
@@ -2,9 +2,11 @@
 
 const crypto = require("crypto");
 const isemail = require("isemail");
-const HIBP = require("../hibp");
+
 const DB = require("../db/DB");
+const HIBP = require("../hibp");
 const EmailUtils = require("../email-utils");
+const ERRORS = require("../errors");
```

Comment:
Stupid idea I'll just throw out there for conversation sake, but...

Should we just dump the `ERRORS` object into `app.locals` so it only needs to be `require()`d once [in server.js]?

https://github.com/mozilla/blurts-server/blob/3e5d511f862eeaf0e560559cc97c768de41dc8bc/server.js#L92-L94

Then we could just use the slightly more verbosier `req.app.locals.ERRORS[code].message`.